scripts/index.js

```diff
@@ -1 +1,21 @@
-console.log('coucou')
+//Need server address filled in URL
+function createNewPost(title, content) {
+    const url = "server address";
+    const date = {title, content};
+
+    fetch(url, {
+        method: "POST",
+        headers: {"Content-Type": application/JSON},
+        body: JSON.stringify(data)})
```

Comment:
**data**  will be undefined . since you are creating a **date** variable in the 4th line.